### PR TITLE
fix camel case on bestscore and highscore

### PR DESCRIPTION
**Pull request by mbburch**
#### What's this PR do?
It changes inconsistent variable names to camel case, so they match JS styling.

#### Where should the reviewer start?
Start in the runner. Song file is also affected.

#### How should this be manually tested?
Please make sure I didn't miss any variables. I think I caught them all, and the game is still working. 

#### Any background context you want to provide?
I think it's a pretty simple change. No context needed.

#### What gif best describes this PR or how it makes you feel?
giphy

### CODE REVIEW

**Comment by mbburch** [comment]
@russelleh @rrgayhart

**Comment by rrgayhart** [comment]
Looks like there are some merge conflicts with this branch that need to be fixed up!

File: index.html [hunk 1]
```diff
@@ -33,7 +33,7 @@
         <button id="song-2">Hard</button>
         <div class="scoreboard">
           <h1 id="score">Score: </h1><br>
-          <h1 id="highscore">High Score: </h1>
+          <h1 id="highScore">High Score: </h1>
```
Comment (rrgayhart) [inline index.html]: I have some concerns about actually changing the CSS selectors to camelcase. Check out the discussion in this [StackOverflow](http://stackoverflow.com/questions/1547986/can-i-use-camel-case-in-css-class-names). Up to you though!

**Comment by rrgayhart** [comment]
Excellent use of .gifs. Good PR - Check out my comment on the CSS selector and fix the mergies and should be good to :ship:

File: index.html (hunk 1 above)
Comment (mbburch) [inline index.html]: Thanks for that article! Good to know. I actually didn't mean to change that one, so good catch!

**Pull request merged**